### ISSUE TITLE
Output json doesn't show when a chart is on multiple repository

### ISSUE THREAD

**Issue by gerchardon** [fabmation-gmbh/helm-whatup#33]
```bash
$ helm whatup
NAME                    consul
NAMESPACE               default
INSTALLED VERSION       0.21.0

REPOSITORY      DEPRECATED      CHART VERSION   APP VERSION     UPDATED                                
stable          true            3.9.6           1.5.3           2020-10-22 12:29:11.284557396 +0000 UTC
incubator       true            0.1.5           0.0.1           2020-10-22 11:31:21.422568698 +0000 UTC
hashicorp       false           0.25.0          1.8.4           2020-10-12 23:25:03.70997427 +0000 UTC 
bitnami         false           8.0.2           1.8.4           2020-10-21 14:56:59.580811175 +0000 UTC
```

and with `-o json`:
```bash
$ helm whatup -o json
[]
```

**Comment by l0nax** [comment]
I will fix this ASAP.

**Comment by l0nax** [comment]
How important/ critical is this feature for you @gerchardon?
Because I have to prioritize some of my issues.

**Comment by gerchardon** [comment]
Not so much with table I can use it but i use jq to show/filter better result

**Comment by l0nax** [comment]
Fixed with version [v0.4.3](https://github.com/fabmation-gmbh/helm-whatup/releases/tag/v0.4.3).
